tests/console/bash.pm

```diff
@@ -0,0 +1,80 @@
+# SUSE's openQA tests
+#
+# Copyright (C) 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+# Summary: Bash regression test (https://progress.opensuse.org/issues/50747)
+# Maintainer: Nalmpantis Orestis (onalmpantis)
+ 
+use base "consoletest";
+use strict;
+use testapi;
+use utils 'zypper_call';
+use version_utils 'is_sle'; 
+
+sub run {
+    # Declaring repo and database variables that change depending on the product tested	
+    my $repo="";
```

Comment:
At the time of writing the code as I remember it was mandatory because there was some kind of error if it didnt happen

Reply:
Interesting topic even if it is not a big issue in my opinion. The variable would be `undef` by default of course. So there could be a case where you distinguish `""` from `undef`
But you might also just check `if ($repo)` and it's all the same to you
If you eg. `print` something, you'll get warnings if it's `undef` 
I'd recommend one of two options 1) `$repo = ''` 2) `print $repo // 'default'`
if it's undef you do the // to give it a fallback

Anyhow, I would not hold a PR because of this :)

Reply:
As I said, it is a cosmetic one. There could be a case mentioned by you, but there is not. The bigger problem in my opinion is the very existence of this `$repo` variable as mentioned below.